### PR TITLE
dev-python/pyperclip: add 1.9.0

### PR DESCRIPTION
**Pull request by henri-gasc**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Review by mgorny — CHANGES_REQUESTED** [review]
Don't add the new version with stable keywords and don't remove the old version.

Also, I'd really like to know whether that behavior change was intentional.

**Comment by henri-gasc** [comment]
If you are talking about the tests, I would the change was intentional but the tests were not updated. And as we don't know what to replace them with, I got those changes

**Comment by mgorny** [comment]
What I'd really like to see is a changelog entry or a commit message, anything proving that it was intentional and not an accidental side effect.

**Comment by henri-gasc** [comment]
Based on commit [5aef21c](https://github.com/asweigart/pyperclip/commit/5aef21c14cda3a9c858adf108581fd2849c6321f) (specifically [lines 550 to 554](https://github.com/asweigart/pyperclip/commit/5aef21c14cda3a9c858adf108581fd2849c6321f#diff-88aa8f803f98776cd1ad3c9fa1172fbe53b6f578e39a22b54478ed37e9a53133R554 ), I believe it is due to the update of PyQt4 to PyQt5

**Comment by eli-schwartz** [comment]
@henri-gasc please *rebase* your changes, and have one commit per logical change you are making. Do not perform an action in one commit, and then roll back to the previous state in a second commit -- this is a sign that the first commit was flawed and needs to be fixed via `git commit --amend`.

To head this off at the pass, it is Gentoo policy to treat the addition of a new version (1.9.0) and dropping of an old version (1.8.2) as *two* logical changes, and have one commit for each. This makes it easier to cherry-pick a single change, revert if something went wrong, etc.

File: dev-python/pyperclip/pyperclip-1.9.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+HOMEPAGE="
+	https://github.com/asweigart/pyperclip/
+	https://pypi.org/project/pyperclip/
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~mips ~ppc64 ~riscv ~s390 ~sparc ~x86"
+
+PATCHES="${FILESDIR}/${P}-fix-test.patch"
```
Comment (mgorny) [inline dev-python/pyperclip/pyperclip-1.9.0.ebuild:21]: `PATCHES` is an array.

Comment (henri-gasc) [reply dev-python/pyperclip/pyperclip-1.9.0.ebuild:21]: Oh yeah, f*ck

File: dev-python/pyperclip/pyperclip-1.9.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	https://pypi.org/project/pyperclip/
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~mips ~ppc64 ~riscv ~s390 ~sparc ~x86"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-test.patch"
+)
```
Comment (mgorny) [inline dev-python/pyperclip/pyperclip-1.9.0.ebuild:23]: Oh, and it goes into `src_prepare` if it's defined. But I'll fix that and merge, don't worry.

**Pull request closed without merging**